### PR TITLE
target all added

### PR DESCRIPTION
**Pull request by Madhan-SWE**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Target `all` added in the makefile. Which could be helpful in running `prow` jobs.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,9 @@ PLATFORM=linux/ppc64le
 
 .EXPORT_ALL_VARIABLES:
 
+.PHONY: all
+all: verify test bin/ibm-powervs-block-csi-driver image
```
Comment (mkumatag) [inline Makefile:34]: but where do we use? do you want to running everything in one shot? Its good to run these things in parallel for better error reporting,

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Madhan-SWE) [reply Makefile:34]: During the presubmit build from prow, ibm-vpc-block-csi-driver runs all these targets.
If not required, we may run only target build.

Comment (mkumatag) [reply Makefile:34]: I feel running them individual will be a better idea I feel to better reporting and coverage, e.g: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1543.

**Comment by mkumatag** [comment]
Just to check how presubmit jobs behaves
/close

File: Makefile (hunk 1 above)
Comment (Madhan-SWE) [reply Makefile:34]: Makes Sense. PR can be closed.

**Pull request closed without merging**